### DISCUSSION TITLE
Classification Banner Question

### DISCUSSION THREAD

**Discussion by kyletracy94** [RocketCommunicationsInc/astro#1318] [Q&A]
Hello team,

I had a question about the RuxClassificationComponent banner.

Our team is utilizing the classification banner, however we have a different classification value than what is available in the current list. Is there a way to customize the classification value (aside from the label option). 

Thanks in advance for your help!

**Comment by dmcalester** [comment]
Hi @kyletracy94,

Can you provide some additional information about what the classification value is? Would it be something like the distribution level or is it a completely different classification? If public GitHub isn’t appropriate please feel free to reach out to support@astrouxds.com
